src/dams_mcda/server.R

```diff
@@ -1496,12 +1513,13 @@ server <- function(input, output, session) {
 		# (d) has three graphs for each dam
 		# d1
 		plotA <- renderPlot2D(
-			t(ResultsMatrix[,,damId]),
+			t(round(ResultsMatrix[,,damId],0)),
 			"D 1", # title
 			alternative_names, # x_labels
 			criteria_names, # y_labels
 			"Total MCDA Score", # y axis label
-			"Decision Criteria:", # legend label
+			"Decision Criteria", # legend label
+			"", # no legend label

```

Comment:
I'm removing legend labels in my branch, up to you whether to do it now or just wait

Reply:
yeah forgot to update the comment, for this
			"Decision Criteria", # y label
we want the empty string as legend label